Content.Server/Chat/Systems/ChatSystem.cs

```diff
@@ -187,7 +187,8 @@ public void TrySendInGameICMessage(
         string? nameOverride = null,
         bool checkRadioPrefix = true,
         bool ignoreActionBlocker = false,
-        LanguagePrototype? languageOverride = null
+        LanguagePrototype? languageOverride = null,
+        string? color = null
```

Comment:
Why are we introducing a color parameter, that is only used in 2 out of 7 code branches, that is only ever has the value of SubtleOOCColor..? It could be referenced statically or even embedded in the .ftl file...